src/main/scala/petclinic/models/Owner.scala

```diff
@@ -0,0 +1,36 @@
+package petclinic.models
+
+import zio._
+import java.util.UUID
+
+
+
+// wraps UUID as a specific OwnerId so that we cannot use an incorrect UUID
+final case class OwnerId(id: UUID) extends AnyVal
+
+object OwnerId {
+  def random: ZIO[Random, Nothing, OwnerId] = Random.nextUUID.map(OwnerId(_))
+}
+
+
+// extracts address to reduce the number of fields in owner
+final case class Address(ownerId: OwnerId, street: String, city: String, state: String, zip: String)
+
+object Address {
+  def make(ownerId: OwnerId, street: String, city: String, state: String, zip: String): Address =
```

Comment:
If we're not going to implementation any additional logic here in the future this could just be `apply`.

Reply:
> If we're not going to implementation any additional logic here in the future this could just be `apply`.

Makes sense. I fixed it.